toc/rfc/rfc-draft-ssh-over-ws.md

```diff
@@ -0,0 +1,269 @@
+# Meta
+[meta]: #meta
+- Name: CF SSH over WebSockets
+- Start Date: 2024-04-08
+- Author(s): @domdom82
+- Status: Draft
+- RFC Pull Request: https://github.com/cloudfoundry/community/pull/807
+
+
+## Summary
+
+This RFC aims at providing an additional way of using CF SSH in an effort to reduce the potential attack surface against SSH.
+
+By tunneling SSH through a WebSocket connection, the SSH client may reuse the same port as regular HTTP traffic, removing the need for port 2222.
+
+## Problem
+
+Cloud Foundry is a shared platform.
+
+In production deployments of Cloud Foundry, it is commonplace for customers to run security scans on their workloads to ensure compliance and reduce attack surfaces.
+
+These scans usually complain that besides port 443 for HTTP traffic, also port 2222 is open for SSH. Many customers demand closure of the port as they
```

Comment:
I am not sure if it is worth mentioning that the ssh is not enabled by default. Manual action is required in order to allow ssh, sadly most of the customers later forget to close it and execute scans.

Reply:
Added more clarity in the document.